stdlib/public/core/StringGuts.swift

```diff
@@ -159,9 +159,9 @@ extension _StringGuts {
   }
 
   @inlinable @inline(__always)
-  internal func withFastUTF8<R>(
-    _ f: (UnsafeBufferPointer<UInt8>) throws -> R
-  ) rethrows -> R {
+  internal func withFastUTF8<Success, Failure: Error>(
+    _ f: (UnsafeBufferPointer<UInt8>) throws(Failure) -> Success
+  ) throws(Failure) -> Success {
```

Comment:
Again, beware that the old routine was `@inlinable`, so it exported an ABI symbol. This change removes that old symbol and replaces it with a new one -- we usually cannot do that. (`@inline(__always)` does not mitigate this.)

Reply:
Thanks for pointing that out. What's the recommended way to mitigate that here?